frontend/src/components/NoMatch.tsx

```diff
@@ -0,0 +1,47 @@
+import {Box, Link} from '@cloudscape-design/components'
+import React from 'react'
+import TopBar from './TopBar'
+import errorPage from './../../public/img/error_pages_illustration.svg'
+import Image from 'next/image'
+import {useTranslation} from 'react-i18next'
+import {useLocationChangeLog} from '../navigation/useLocationChangeLog'
+
+export function NoMatch() {
+  const {t} = useTranslation()
+  useLocationChangeLog()
+
+  return (
+    <>
+      <TopBar />
+      <div
+        style={{
+          position: 'absolute',
+          left: '50%',
+          top: '50%',
+          transform: 'translate(-50%, -50%)',
+        }}
+      >
+        <Box>
+          <Image src={errorPage} alt="test" height={100} />
+          <h1>{t('noMatch.title')}</h1>
+          <p>{t('noMatch.description')}</p>
+          <p>{t('noMatch.links')}</p>
+          <ul>
+            <li>
+              <Link href="/">{t('noMatch.home')}</Link>
+            </li>
+            <li>
+              <Link
+                external
+                externalIconAriaLabel={t('global.openNewTab')}
+                href={t('global.docs.link')}
+              >
+                {t('global.docs.title')}
+              </Link>
+            </li>
+          </ul>
+        </Box>
```

Comment:
issue: for consistency of typography, see if you can use components provided by the design system. For instance the [TextContent component](https://cloudscape.design/components/text-content/?tabId=playground) seems a good choice here

Reply:
Right, thanks!